docs/item_template.md

```diff
@@ -2336,7 +2336,7 @@ Bitmask field that contains flags that the item has on it. As all other such fie
 <tr class="even">
 <td><p>134217728</p></td>
 <td>0x08000000</td>
-<td><p>Bind to Account (Set Quality = 7 for the corresponding color. Might require to set Bonding = 1)</p></td>
+<td><p>Bind to Account (Requires to set Bonding > 0)</p></td>
```

Comment:
why removing the tip about the color?

Reply:
see my previous PR (it should have been in previous PR)

Reply:
see here https://github.com/azerothcore/wiki/pull/220

Reply:
Are you sure that you can still have Bind to Account without Q = 7? Never actually tested it myself

Reply:
yes and many items are like that by default (example: murloc costume)

Reply:
Ah yes that is true, now I remember